RELEASING.md

```diff
@@ -127,6 +127,22 @@ be a release candidate.
         ... etc ...
    ```
 
+1. Update the default version tags used by images (stored in `_includes/vX.Y/charts/calico/values.yaml`) for the helm chart from `master` to `release-vX.Y`. 
```

Comment:
Grr. I know this is probably right, but it feels pretty annoying to have to update this on each release. I'd like to start removing manual steps from here with automated ones.

Can you maybe automate this as part of a make target (e.g., see `make update-authors`) or even `do_release.py` so it's at least not that much typing?

e.g.

```
make RELEASE_STREAM=vX.Y update-values-yaml
```

or something like that?

Reply:
Having thought about it more, I think some major changes to `versions.yml` could circumvent this issue as we could  use `_versions.yml` as `versions.yml` so that the info isn't propogated into two places.

I'll hold off on this merge until I have a better proposal together on the matter